qiskit_experiments/library/randomized_benchmarking/rb_utils.py

```diff
@@ -294,3 +310,264 @@ def calculate_2q_epg(
+        "rz": 0.0,
+        "id": 0.0,
+        "h": 1.0,
+    }
+
+    # Gate count in (CX, CSX)-based decomposition, 1q gate contribution is ignored.
+    # Amplitude or duration modulated pulse implementation is not considered.
+    standard_2q_ratio = {
+        "swap": 3.0,
+        "rxx": 2.0,
+        "rzz": 2.0,
+        "cx": 1.0,
+        "cy": 1.0,
+        "cz": 1.0,
+        "ch": 1.0,
+        "crx": 2.0,
+        "cry": 2.0,
+        "crz": 2.0,
+        "csx": 1.0,
+        "cu1": 2.0,
+        "cp": 2.0,
+        "cu": 2.0,
+        "cu3": 2.0,
+    }
+
+    if n_qubits == 1:
+        return standard_1q_ratio.get(gate, None)
+
+    if n_qubits == 2:
+        return standard_2q_ratio.get(gate, None)
+
+    raise QiskitError(
+        f"Standard gate error ratio for {n_qubits} qubit RB is not provided. "
+        "Please explicitly set 'gate_error_ratio' option of the experiment."
+    )
+
+
+def calculate_epg(
+    epc: Union[float, uncertainties.UFloat],
+    qubits: Sequence[int],
+    gate_error_ratio: Dict[str, float],
+    gate_counts_per_clifford: Dict[QubitGateTuple, float],
+) -> Dict[str, Union[float, uncertainties.UFloat]]:
+    r"""Compute EPGs of basis gates from fit EPC value.
+
+    This function just distributes the measured EPC :math:`\cal E` into basis gates :math:`\{g_i\}`
+    defined in the ``gate_error_ratio`` dictionary according to the ratio
+    and gate counts.
+
+    Given we have :math:`n_i` gates with independent error :math:`e_i` per Clifford,
+    the total EPC is estimated by the composition of error from every basis gate,
+
+    .. math::
+
+        {\cal E} = 1 - \prod_{i} (1 - e_i)^{n_i} \sim \sum_{i} n_i e_i + O(e^2)
+
+    where :math:`e_i \ll 1` and the higher order terms can be ignored.
+    We cannot distinguish :math:`e_i` with single EPC value here,
+    however by defining an error ratio :math:`r_i` with respect to
+    some standard value :math:`e_0`, we can compute EPG :math:`e_i` for each basis gate.
+
+    .. math::
+
+        {\cal E} \sim e_0 \sum_{i} n_i r_i
+
+    The EPG of :math:`i` th basis gate will be
+
+    .. math::
+
+        e_i \sim r_i e_0 = \dfrac{r_i{\cal E}}{\sum_{i} n_i r_i}.
+
+    The gate errors retuned by this function is computed based on such simple assumption,
+    this is not necessary representing actual gate error on the hardware.
+    If you have multiple kinds of basis gates with unclear error ratio :math:`r_i`,
+    :class:`InterleavedRB` experiment will give you accurate error value :math:`e_i`.
+
+    Args:
+        epc: Error per Clifford.
+        qubits: List of qubits used in the experiment.
+        gate_error_ratio: A dictionary of assumed ratio of errors among basis gates.
+        gate_counts_per_clifford: Basis gate counts per Clifford gate.
+
+    Returns:
+        A dictionary of gate errors keyed on the gate name.
+    """
+    norm = 0
+    for gate, r_epg in gate_error_ratio.items():
+        key = QubitGateTuple(*qubits, gate=gate)
```

Comment:
Why not just do `key = (tuple(sorted(qubits)), gate)` and avoid adding a new class?